deployment/azure.md

```diff
@@ -68,3 +68,26 @@ az containerapp create \
 ```
 
 4. Visit the url printed from the output of the previous command.
+
+##Persistent Disk in Container Apps
```

Comment:
I am voting to not to accept this solution as it is unstable due to the nature of sqlite (that is designed to not to be a database for concurrent users) and poor integration with File Share. It sometimes saves the data, sometimes not, sometimes is able to read and sometimes not.

I am now testing the solution with a file share as store (without sqllite), similar as it is done in Render.